### ISSUE TITLE
Is fft window size=5second, and a sample's time = 100 second?

### ISSUE THREAD

**Issue by kaixinbuyu** [yscacaca/HHAR-Data-Process#4]
1. Im not clear **is the "WIDE" means lstm step size = 20 = 100senconds?** 
Is a trainning sample with all steps lasted for 20*5sec = 100 senconds? 

a fft window :                       : lasted for **5 sencond**         (fftSpan = 5.);
a sample = 20 * fft-window  : lasted for **100 sencond**     (wide = 20, is wide means lstm step?)

2.in DeepSence's training code:
                # inputs shape (BATCH_SIZE, WIDE, FEATURE_DIM)
		sensor_inputs = tf.expand_dims(inputs, axis=3)
		# sensor_inputs shape (BATCH_SIZE, **WIDE**, FEATURE_DIM, CHANNEL=1)
		acc_inputs, gyro_inputs = tf.split(sensor_inputs, num_or_size_splits=2, axis=2)
                acc_conv1 = layers.convolution2d(acc_inputs...
                #acc_inputs 's shape = (64, 20, 60, 1)

So the 2d conv extract features from:
 (1) dim-sensor:(x,y,z,60 features),and 
 (2) dim-?: 20 
**is 20=WIDE means lstm steps**??if that why the first conv use all the time information?this is not followd in the paper way.

**or is it just means d in "d x 2f" caused by fft**? 
as the code:"accInterpTime = np.linspace(0.0, fftSpan*1, **sepcturalSamples***1)" sepcturalSamples=10

I'm confused that is the code style as same as this acticle:
"but actually we’re going to process slice by slice in the T dimension (one window at a time). **Each **d x 2f** window slice is passed through a convolution neural network** (**the conv above**) component comprising three stages as illustrated below"
[https://adriancolyer.files.wordpress.com/2017/05/deepsense-sketch-5.jpeg?w=480](https://adriancolyer.files.wordpress.com/2017/05/deepsense-sketch-5.jpeg?w=480)

================================================
Last night I read the paper and code again,I figured out that:

WIDE = 20 cames from fft;
lstm steps cames from :
 "used = tf.sign(tf.reduce_max(tf.abs(inputs), reduction_indices=2)) #(BATCH_SIZE, WIDE)"
The parameter's short name is a little confusing for beginners.

================================================

**Comment by hsm207** [comment]
@kaixinbuyu so is the input to the network:

1. a 5-second sample that is split into 20 intervals of length 0.25 second each, or
2. a 100-second sample split that is split into 20 intervals of 5 second each ?

Thank you for your help.
